qiskit/circuit/library/generalized_gates/qft.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""QFTGate: gate class for natively reasoning about Quantum Fourier Transforms."""
+
+from __future__ import annotations
+import numpy as np
+from qiskit.circuit.quantumcircuit import Gate
+
+
+class QFTGate(Gate):
+    r"""Quantum Fourier Transform Circuit.
+
+    The Quantum Fourier Transform (QFT) on :math:`n` qubits is the operation
+
+    .. math::
+
+        |j\rangle \mapsto \frac{1}{2^{n/2}} \sum_{k=0}^{2^n - 1} e^{2\pi ijk / 2^n} |k\rangle
+
+    """
+
+    def __init__(
+        self,
+        num_qubits: int,
+    ):
+        """Construct a new QFT gate.
+
```

Comment:
I think in most places we don't put this initial sentence anymore if it doesn't provide any info, since it looks nicer in the docs if we just start with the `Args` 🙂 but that's certainly true in the circuit lib 🙂 
```suggestion
        """
```

Reply:
Done.